src/main/java/com/esri/core/geometry/SizeOf.java

```diff
@@ -0,0 +1,104 @@
+package com.esri.core.geometry;
```

Comment:
@mbasmanova Could you add a copyright header here? See here Copyright and Licensing https://github.com/esri/contributing.